lib/plugins/aws/provider.js

```diff
@@ -757,6 +768,33 @@ class AwsProvider {
             },
             iamManagedPolicies: { type: 'array', items: { $ref: '#/definitions/awsArnString' } },
             iamRoleStatements: { $ref: '#/definitions/awsIamPolicyStatements' },
+            ecr: {
+              type: 'object',
+              properties: {
+                images: {
+                  type: 'object',
+                  patternProperties: {
+                    '^[a-z]{1,32}$': {
```

Comment:
I assume it's case insensitve. For the sake of clarity, I would probably add `A-Z` also I think we should support numbers and `-` (just not as fist chars), what do you think?

Reply:
it's in fact case sensitive and you cannot use uppercase. I totally agree that we can use numbers and additional chars though :+1: